drivers/dma/dma_iproc_pax_common.h

```diff
@@ -0,0 +1,536 @@
+#define AE_INTERFACE_GROUP_5_INTERRUPT_MASK 0x308c
+#define AE_INTERFACE_GROUP_5_INTERRUPT_CLEAR 0x3090
+#define AE_INTERFACE_GROUP_6_INTERRUPT_MASK 0x3094
+#define AE_INTERFACE_GROUP_6_INTERRUPT_CLEAR 0x3098
+#define AE_INTERFACE_GROUP_7_INTERRUPT_MASK 0x309c
+#define AE_INTERFACE_GROUP_7_INTERRUPT_CLEAR 0x30a0
+#define AE_INTERFACE_TOP_INTERRUPT_STATUS_MASK 0x30a4
+#define AE_INTERFACE_TOP_INTERRUPT_STATUS_CLEAR 0x30a8
+#define REORDER_INTERRUPT_STATUS_MASK 0x30ac
+#define REORDER_INTERRUPT_STATUS_CLEAR 0x30b0
+#define DME_INTERRUPT_STATUS_MASK 0x30b4
+#define DME_INTERRUPT_STATUS_CLEAR 0x30b8
+#define REORDER_FIFO_PROG_THRESHOLD 0x30bc
+#define GROUP_PKT_EXTENSION_SUPPORT 0x30c0
+#define RM_GENERAL_MSI_DATA_VALUE 0x30c4
+#define RM_AXI_READ_BURST_THRESHOLD 0x30c8
+#define RM_GROUP_RING_COUNT 0x30cc
+#define MSI_INTERRUPT_STATUS_MASK 0x30d0
+#define MSI_INTERRUPT_STATUS_CLEAR 0x30d4
+#define RM_MSI_DISABLE 0x30d8
+#define RM_RESERVE 0x30fc
+#define RM_RING_FLUSH_STATUS 0x3100
+#define RING_SEQUENCE_NUMBER_OVERFLOW 0x3104
+#define AE_SEQUENCE_NUMBER_OVERFLOW 0x3108
+#define MAX_SEQUENCE_NUMBER_FOR_ANY_RING 0x310c
+#define MAX_SEQUENCE_NUMBER_ON_MONITOR_RING 0x3110
+#define MAX_SEQUENCE_NUMBER_ON_ANY_AE 0x3114
+#define MAX_SEQUENCE_NUMBER_ON_MONITOR_AE 0x3118
+#define MIN_MAX_LATENCY_MONITOR_RING_TOGGLE 0x311c
+#define MIN_MAX_LATENCY_MONITOR_RING_ADDRESSGEN 0x3120
+#define Ring_Activity 0x3124
+#define AE_Activity 0x3128
+#define RM_MAIN_HW_INIT_DONE 0x312c
+#define RM_MEMORY_POWER_STATUS 0x3130
+#define CONFIG_STATUS_0 0x3134
+#define CONFIG_STATUS_1 0x3138
+#define TOGGLE_STATUS_0 0x313c
+#define TOGGLE_STATUS_1 0x3140
+#define DDR_ADDR_GEN_STATUS_0 0x3144
+#define DDR_ADDR_GEN_STATUS_1 0x3148
+#define PACKET_ALIGNMENT_STATUS_0 0x314c
+#define PACKET_ALIGNMENT_STATUS_1 0x3150
+#define PACKET_ALIGNMENT_STATUS_2 0x3154
+#define PACKET_ALIGNMENT_STATUS_3 0x3158
+#define AE_INTERFACE_GROUP_0_STATUS_0 0x315c
+#define AE_INTERFACE_GROUP_0_STATUS_1 0x3160
+#define AE_INTERFACE_GROUP_1_STATUS_0 0x3164
+#define AE_INTERFACE_GROUP_1_STATUS_1 0x3168
+#define AE_INTERFACE_GROUP_2_STATUS_0 0x316c
+#define AE_INTERFACE_GROUP_2_STATUS_1 0x3170
+#define AE_INTERFACE_GROUP_3_STATUS_0 0x3174
+#define AE_INTERFACE_GROUP_3_STATUS_1 0x3178
+#define AE_INTERFACE_GROUP_4_STATUS_0 0x317c
+#define AE_INTERFACE_GROUP_4_STATUS_1 0x3180
+#define AE_INTERFACE_GROUP_5_STATUS_0 0x3184
+#define AE_INTERFACE_GROUP_5_STATUS_1 0x3188
+#define AE_INTERFACE_GROUP_6_STATUS_0 0x318c
+#define AE_INTERFACE_GROUP_6_STATUS_1 0x3190
+#define AE_INTERFACE_GROUP_7_STATUS_0 0x3194
+#define AE_INTERFACE_GROUP_7_STATUS_1 0x3198
+#define AE_INTERFACE_TOP_STATUS_0 0x319c
+#define AE_INTERFACE_TOP_STATUS_1 0x31a0
+#define RM_REORDER_STATUS_0 0x31a4
+#define RM_REORDER_STATUS_1 0x31a8
+#define RM_REORDER_STATUS_2 0x31ac
+#define RM_REORDER_STATUS_3 0x31b0
+#define RM_REORDER_STATUS_4 0x31b4
+#define RM_REORDER_STATUS_5 0x31b8
+#define CONFIG_INTERRUPT_STATUS 0x31bc
```

Comment:
Lack of prefix here is a good example, since this now looks like a Kconfig define (CONFIG_INTERRUPT_STATUS), which is confusing.

Reply:
I understand, let me fix it new patchset.